### DISCUSSION TITLE
Macbook 2017 NonTouch SSD Sabrent 1 TB

### DISCUSSION THREAD

**Discussion by Sasch0w** [kvic-z/SsdPmEnabler#22] [Q&A]
Hello Kvic,

first of all thank you for your efforts and work on the battery loss problem. I have upgraded my Macbook 2017 with a Sabrent 1TB. Unfortunately, my Macbook consumes 5 times more power when idle. I have installed the SsdPmEnabler as described. The message in the terminal is the right one, but in the System Information.app is the error message. What can I do? Kind regard

<img width="1437" alt="Bildschirmfoto 2022-04-15 um 10 11 09" src="https://user-images.githubusercontent.com/103735205/163547183-82ea5813-c868-44c9-b564-fd5edd64435d.png">
s

**Comment by kvic-z** [comment]
Hello,

In lack of details, I'm doing some guesswork here. A typical Apple SSD idling around 0.03A. Five times worse for your 1TB "dark blue" Sabrent SSD (based on Phison E12 chipset) is around 0.15A which is about a typical Phison E12 based solution would do.

So it seems ssdpmEnabler didn't do its wonder here even though kernel log seemed to show it did get loaded properly. I've heard from another user who said he didn't see a difference with/without kext. So that was what it's and I haven't kept up with latest updates from other users. You may want to check the thread yourself (https://forums.macrumors.com/threads/found-this-adaptor-for-2016-2017-non-touch-bar-for-ssd-upgrades.2231881/page-17). See if people have better luck with a specific make of SSDs or other workarounds.

Cheers
